hazelcast-integration/spring-jcache/src/main/java/com/hazelcast/spring/jcache/SpringJCacheJavaConfig.java

```diff
@@ -0,0 +1,24 @@
+package com.hazelcast.spring.jcache;
+
+import com.hazelcast.client.HazelcastClient;
+import com.hazelcast.core.Hazelcast;
+import org.springframework.context.ApplicationContext;
+import org.springframework.context.annotation.AnnotationConfigApplicationContext;
+
+import static com.hazelcast.spring.jcache.DummyBeanTest.*;
+
+/**
+ * Example of Spring Caching with JCache
+ */
+public class SpringJCacheJavaConfig {
+    public static void main(String[] args) {
+        System.setProperty("hazelcast.jcache.provider.type", "client");
+        ApplicationContext applicationContext =
+            new AnnotationConfigApplicationContext(AppConfig.class);
+
+        calculateInvocationTime(applicationContext);
```

Comment:
This method does not only measure time but also does the actual method invocations. I think this is a little misleading. It is hard to understand where actual calls to entity object is done at first look.

If you want to reuse the code inside `calculatInvocationLogic` please change its name to something like `doInvocations` as that is what is important for code sample although I prefer having application logic here rather than delegating the work to another static method.